tests/unit/contentRecog/test_contentRecog.py

```diff
@@ -0,0 +1,155 @@
+#tests/unit/contentRecog/test_contentRecog.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2017 NV Access Limited
+
+"""Unit tests for the contentRecog module.
+"""
+
+import unittest
+import contentRecog
+import textInfos
+
+class TestResultCoordConverter(unittest.TestCase):
+
+	def test_noOffsetNoResize(self):
+		conv = contentRecog.ResultCoordConverter(0, 0, 1)
+		actual = conv.convertX(100), conv.convertY(200)
+		self.assertEqual(actual, (100, 200))
+
+	def test_withOffsetNoResize(self):
+		conv = contentRecog.ResultCoordConverter(10, 20, 1)
+		actual = conv.convertX(100), conv.convertY(200)
+		self.assertEqual(actual, (110, 220))
+
+	def test_noOffsetWithResize(self):
+		conv = contentRecog.ResultCoordConverter(0, 0, 2)

```

Comment:
As discussed its a little confusing to me that 2 results in a halving of the coords.